portal/jam-embedded-api.md

```diff
@@ -14,46 +14,46 @@ Fiddler Jam Embedded is a JavaScript library that provides an API to integrate t
-                });
-                // .. custom implementation follows here
-            </script>
-        </head>
-    </html>        
+1. Initialize the Fiddler Jam Embedded object with your Fiddler Jam API key. The Fiddler Jam Embedded object is attached to the [`window`](https://www.w3schools.com/jsref/obj_window.asp) object of the page as a property called `_fiddlerJamEmbedded`. Use the `load` event of the Fiddler Jam Embedded object to initialize the Jam API and to add your custom implementation.
+
+    ```html
+        <html lang="en">
+            <head>
+                <meta charset='utf-8'>
+                <meta name='viewport' content='width=device-width,initial-scale=1'>
+            
+                <title>Fiddler Jam Embedded</title>
+
+                <script src="https://downloads.getfiddler.com/jam-embedded/fiddler-jam-embedded.js" id="jamEmbeddedScript" async></script>
+                <script>
+                    const jamEmbeddedScript = document.getElementById('jamEmbeddedScript');
+                    jamEmbeddedScript.crossOrigin = 'anonymous';  // set crossOrigin to enable CORS (and to be able to use the load event below)
+                    jamEmbeddedScript.addEventListener('load', () => {
+                        const jam = window['_fiddlerJamEmbedded'];
+                        jam.init({
+                            apiKey: 'API_KEY'
+                        });
+
+                        // custom implementation follows here
+                    });
+                </script>
+            </head>
+        </html>        
     ```
 
-    >important The Fiddler Jam API key is unique per organization and grants access to your workspaces.
+    >important The Fiddler Jam API key is unique per organization and grants access to your organizational workspaces.
 
 1. Create a `service-worker.js` file and import the service worker script from the following CDN link: https://downloads.getfiddler.com/jam-embedded/fje-service-worker.js.
 
-    By default, the `service-worker.js` file is expected to be on the same level as the HTML page that contains the `fiddler-jam-embedded.js` CDN script. You can change the default path of the worker file through the `InitOptions` object and its `serviceWorkerPath` property. Below you can find an example for importing the service worker from the Fiddler CDN.
+    By default, the `service-worker.js` file is expected to be on the same level as the HTML page that contains the `fiddler-jam-embedded.js` CDN script. You can change the default path of the worker file through the `InitOptions` object and its `serviceWorkerPath` property. Below is an example of importing the service worker from the Fiddler CDN.
     
-    ```JavaScript
-    self.importScripts(`https://downloads.getfiddler.com/jam-embedded/fje-service-worker.js`);
-    ```
+```JavaScript
+self.importScripts(`https://downloads.getfiddler.com/jam-embedded/fje-service-worker.js`);
```

Comment:
The format on the test docs site does not look good when ` are used, maybe you can use `"` here
![Screen Shot 2022-07-14 at 9 09 51](https://user-images.githubusercontent.com/8351653/178912275-f1a0114f-2771-4d51-9736-a2fd1bb4c521.png)